.github/dependabot.yml

```diff
@@ -4,13 +4,11 @@ updates:
       directory: "/evm_loader"
       schedule:
           interval: "daily"
-          time: "10:00"
-          timezone: "UTC"
-      target-branch: "develop"
```

Comment:
By default, Dependabot opens pull requests against default branch, which is `develop` for this repo.